tests/fixers/test_django_urls.py

```diff
@@ -320,9 +323,354 @@ def test_complete():
         urlpatterns = [
             path('', views.index, name='index'),
             path('about/', views.about, name='about'),
+            path('post/<slug:slug>/', views.post, name='post'),
             re_path(r'^post/(?P<slug>[w-]+)/$', views.post, name='post'),
             re_path(r'^weblog/', include('blog.urls')),
         ]
         """,
         settings,
     )
+
+
+@pytest.mark.parametrize(
```

Comment:
deliberately didn't use parameterize to save indentation and ensure they're all named, I'm rewriting this

Reply:
Sorry for the inconvenience, I was unsure about the reason behind your choice of not using parametrize. I should have followed the existing style, mb!